databuilder/extractor/minio_extractor.py

```diff
@@ -0,0 +1,92 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import csv
+import importlib
+# from collections import defaultdict
+
+from pyhocon import ConfigTree
+from typing import Any
+import boto3
+
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.models.table_metadata import TableMetadata, ColumnMetadata
+
+class MinioExtractor(Extractor):
+
+    """
+    An Extractor that extracts records via CSV.
+    """
+
+    def init(self, conf: ConfigTree) -> None:
+        """
+        :param conf:
+        """
+        self.conf = conf
```

Comment:
What are we actually loading from the conf? Presumably the bucket name, endpoint url, aws_access_key_id, and aws_secret_access_key should all be configuration points that are passed from the original argparse.